### PR TITLE
Remove warning about mixed mode.

### PR DESCRIPTION
**Pull request by sigurdm**
The warning was hard to parse - and would be triggered too often (too noisy).

We can reconsider this later when we have a better feeling for the migration flow
or most packages are fully migrated so we would have less noise.

Fixes: https://github.com/dart-lang/pub/issues/2717

### CODE REVIEW

**Review by jonasfj — APPROVED** [review]

**Comment by mit-mit** [comment]
@sigurdm looks like this needs a small tweak to remove a few imports:

```
25.88s$ dartanalyzer --fatal-infos --fatal-warnings .
Analyzing pub...
  error • Unused import: 'package:meta/meta.dart'. • lib/src/solver/result.dart:6:8 • unused_import
  error • Unused import: '../log.dart'. • lib/src/solver/result.dart:10:8 • unused_import
  error • Unused import: '../null_safety_analysis.dart'. • lib/src/solver/result.dart:11:8 • unused_import
  error • Unused import: '../system_cache.dart'. • lib/src/solver/result.dart:16:8 • unused_import
4 errors found.
The command "dartanalyzer --fatal-infos --fatal-warnings ." exited with 3.
cache.2
store build cache
```
